test/fixtures/mysql-private/variables.tf

```diff
@@ -23,3 +23,8 @@ variable "network_name" {
   default = "mysql-private"
   type    = string
 }
+
+variable "cloudsql_mysql_sa" {
+  type        = string
+  description = "IAM service account user created for Cloud SQL."
+}
```

Comment:
```suggestion

```
YAGNI, because you make the service account in your test fixture:
```terraform
resource "google_service_account" "cloudsql_mysql_sa" {
  project    = module.project.project_id
  account_id = "cloudsql-mysql-sa-01"
}
```